.github/workflows/mainline-protection.yaml

```diff
@@ -6,13 +6,19 @@
 name: Mainline Protection
 on:
   pull_request:
-    branches:
-      - mainline
+    types: [opened, synchronize, edited, reopened]
 # The purpose of this workflow is to create a failing Status check on pull request against mainline. This will prevent
 # PR from being merged into mainline.
 jobs:
   mainline-protection:
-    name: Only create PR against develop branch, not mainline branch
+    name: Only create PR against develop branches, not mainline branches
```

Comment:
Nitpick: Maybe I'm missing something, but why is it plural `branches`? We only have one `develop` branch and one `mainline` branch.

Reply:
reverted to `branch`. This was leftover from copying from FWoA since we have `mainline` and `smart-mainline` there